Data/ByteString/Lazy/Char8.hs

```diff
@@ -569,6 +571,17 @@ elemIndex :: Char -> ByteString -> Maybe Int64
 elemIndex = L.elemIndex . c2w
 {-# INLINE elemIndex #-}
 
+-- | /O(n)/ The 'elemIndexEnd' function returns the last index of the
+-- element in the given 'ByteString' which is equal to the query
+-- element, or 'Nothing' if there is no such element. The following
+-- holds:
+--
+-- > elemIndexEnd c xs ==
+-- > (-) (length xs - 1) `fmap` elemIndex c (reverse xs)
```

Comment:
I think this expression is pretty hard to read. Maybe something like this would be slightly simpler to parse?
```suggestion
-- > let l = length xs - 1
-- > in (l -) `fmap` elemIndex c (reverse xs)
```

Reply:
Updated. Is it better now?

Reply:
Much better. Thanks!